src/ngraph/pass/fused_op_decomposition.cpp

```diff
@@ -61,7 +66,6 @@ bool pass::FusedOpDecomposition::run_on_node(shared_ptr<Node> node)
         {
             for (size_t j = 0; j < output_node->get_outputs().size(); j++, i++)
             {
-                // TODO: Provenance
```

Comment:
@diyessi @jbobba Do you know something more about this `TODO`? If we still need to add any provenance here?

Reply:
I remember adding provenance during the defusion, so we don't need more.